ladder/teams/README.md

```diff
@@ -0,0 +1,51 @@
+# How Cilium Org Teams Are Managed
+Each Cilium org team that wishes to self-manage its team membership can do so using this directory.
+
+Each team can establish a file in this directory managed by the CodeOwners system with the assistance of 
+the Org admins (or duly deputized members of the org tasked with helping initiating team self-management). 
+
+On merge into the main branch (after appropriate approvals and review) GitHub automation 
+will propogate updates from the team files into the Cilium Organization's team membership using the GitHub API.
+
+## Are You a Team Member or a Mentor? Choose, but Choose Wisely.
+With membership comes both priviledges and responsibilities.  One of the responsibilities of being on an active Cilium team member is 
```

Comment:
Can we link to the contributor ladder here and use the same terminology as there? I think you mean to say active Cilium Reviewer?

Reply:
I can do that

Reply:
I added some preamble at the beginning to try to make it the difference between Project Team and GitHub team clearer.
I need to add the specific link backs to other community repo docs with another commit.